app/static/js/createEvents.js

```diff
@@ -173,7 +162,16 @@ $(document).ready(function () {
     if ($(this).val() && $("#endDatePicker-" + $(this).data("page-location")).val()) {
       calculateRecurringEventFrequency();
     }
-  });
+  })
+  // Time picker click events
+  $(".timeIcons").on("click", function(e) {
+    e.stopPropagation()
+    console.log($(this).siblings(".timepicker").timepicker("closed") );
+    if ($(this).siblings(".timepicker").timepicker("closed"))
+      $(this).siblings(".timepicker").timepicker("open"); 
+    else 
+    $(this).siblings(".timepicker").timepicker("close")
```

Comment:
This should be indented. Also let's use braces around the blocks, even though they are single line

Reply:
fixed formatting